### PR TITLE
Remove BG for taps

### PR DESCRIPTION
**Pull request by nicothin**

### CODE REVIEW

File: dist/sass/style/_tcon-button.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@
   width: $tcon-size;
   background: transparent;
   outline: none;
+  -webkit-tap-highlight-color: rgba(0,0,0,0);
```
Comment (kwarkjes) [inline dist/sass/style/_tcon-button.scss]: could you add the android fallback : `-webkit-tap-highlight-color: transparent; /* For some Androids */` [see phonegap tips](http://phonegap-tips.com/articles/essential-phonegap-css-webkit-tap-highlight-color.html)

Comment (nicothin) [inline dist/sass/style/_tcon-button.scss]: I can add, but Android [supports color rgba](http://caniuse.com/#feat=css3-colors)
this project is supported Android 4+
I do not understand why this is to add :)

Comment (kwarkjes) [inline dist/sass/style/_tcon-button.scss]: See [comment of Brad](http://phonegap-tips.com/articles/essential-phonegap-css-webkit-tap-highlight-color.html#comment-1406207148)

Comment (kwarkjes) [inline dist/sass/style/_tcon-button.scss]: Nevermind it looks like `transparent` is supposed to be the fallback for the browsers that does'nt support rgba value

`-webkit-tap-highlight-color: rgba(0,0,0,0);` should work on all Androids

**Comment by grayghostvisuals** [comment]
Thanks for this PR, but I'm a bit confused still if we need this or not based on the convo.

File: dist/sass/style/_tcon-button.scss (hunk 1 above)
Comment (nicothin) [inline dist/sass/style/_tcon-button.scss]: o...
ok, added.

**Comment by grayghostvisuals** [comment]
@nicothin Ok, so we need this fallback for Android 4 correct?

**Comment by nicothin** [comment]
@grayghostvisuals 
I tested on Android 4.4.2 only (does not require).
but I think we can have this fallback,

**Pull request merged**